src/lib/stitching/exchange/__tests__/schema.test.ts

```diff
@@ -1,41 +1,58 @@
-import { executableExchangeSchema } from "../schema"
 import {
   getTypesFromSchema,
   getRootFieldsFromSchema,
   getMutationFieldsFromSchema,
 } from "lib/stitching/lib/getTypesFromSchema"
-import { printSchema } from "graphql"
+import { getExchangeTransformedSchema } from "./testingUtils"
 
 it("Does not include generic type names", async () => {
-  const exchangeSchema = await executableExchangeSchema()
+  const exchangeSchema = await getExchangeTransformedSchema()
   const exchangeTypes = await getTypesFromSchema(exchangeSchema)
 
-  expect(exchangeTypes).not.toContain("Order")
   expect(exchangeTypes).not.toContain("LineItem")
   expect(exchangeTypes).not.toContain("DateTime")
 
-  expect(exchangeTypes).toContain("EcommerceOrder")
+  expect(exchangeTypes).not.toContain("Order")
+  expect(exchangeTypes).toContain("CommerceOrder")
+})
+
+it("Makes exceptions for Types used by in Eigen queries", async () => {
+  // We want to make sure that old Eigen queries don't break
+  //
+  // See: https://github.com/artsy/eigen/blob/master/Artsy/Networking/create_offer.graphql
+  // and: https://github.com/artsy/eigen/blob/master/Artsy/Networking/create_order.graphql
+  //
+  const exchangeSchema = await getExchangeTransformedSchema()
+
+  const exchangeTypes = await getTypesFromSchema(exchangeSchema)
+  const mutationFields = await getMutationFieldsFromSchema(exchangeSchema)
+
+  expect(exchangeTypes).toContain("OrderWithMutationSuccess")
+  expect(exchangeTypes).toContain("OrderWithMutationFailure")
+  expect(exchangeTypes).toContain("CreateOrderWithArtworkInput")
+  expect(exchangeTypes).toContain("OrderWithMutationSuccess")
+
+  // What we want
+  // expect(mutationFields).toContain("ecommerceCreateOrderWithArtwork")
```

Comment:
I don't fully understand the messaging here-- should this be uncommented if it's `What we want`?

Reply:
Look below, when we switch to the default namespace - lines 40/41 will fail and we can replace with 36/37